test/unit/services/DateService.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { DateService, TimespanResolution } from "../../../src/services/DateService";
+
+type TimeArray = [
+  days: number,
+  hours: number,
+  minutes: number,
+  seconds: number
+];
+type TestData = [
+  resolution: TimespanResolution,
+  timeArray: TimeArray,
+  expected: string
+];
+
+describe("DateService", () => {
+  let dateService: DateService;
+  beforeAll(() => {
+    dateService = new DateService();
+  });
+
+  describe("#ticksToTimeSpanString", () => {
+    /* prettier-ignore */
+    const testData: TestData[] = [
```

Comment:
@dkent600 @IonelLupu 

Could you also look over these test cases?
I went through them, and should be what we want now.

Reply:
yes, looks good